integration-tests/actions/vrf/common/actions.go

```diff
@@ -306,26 +315,29 @@ func CreateVRFKeyOnVRFNode(vrfNode *VRFNode, l zerolog.Logger) (*client.VRFKey,
 	return vrfKey, pubKeyCompressed, nil
 }
 
-func FundNodesIfNeeded(ctx context.Context, existingEnvConfig *vrf_common_config.ExistingEnvConfig, client blockchain.EVMClient, l zerolog.Logger) error {
+func FundNodesIfNeeded(ctx context.Context, existingEnvConfig *vrf_common_config.ExistingEnvConfig, client *seth.Client, l zerolog.Logger) error {
 	if *existingEnvConfig.NodeSendingKeyFundingMin > 0 {
 		for _, sendingKey := range existingEnvConfig.NodeSendingKeys {
 			address := common.HexToAddress(sendingKey)
-			sendingKeyBalance, err := client.BalanceAt(ctx, address)
+			sendingKeyBalance, err := client.Client.BalanceAt(ctx, address, nil)
 			if err != nil {
 				return err
 			}
 			fundingAtLeast := conversions.EtherToWei(big.NewFloat(*existingEnvConfig.NodeSendingKeyFundingMin))
 			fundingToSendWei := new(big.Int).Sub(fundingAtLeast, sendingKeyBalance)
-			fundingToSendEth := conversions.WeiToEther(fundingToSendWei)
 			log := l.Info().
 				Str("Sending Key", sendingKey).
 				Str("Sending Key Current Balance", sendingKeyBalance.String()).
 				Str("Should have at least", fundingAtLeast.String())
 			if fundingToSendWei.Cmp(big.NewInt(0)) == 1 {
 				log.
-					Str("Funding Amount in ETH", fundingToSendEth.String()).
+					Str("Funding Amount in wei", fundingToSendWei.String()).
```

Comment:
why not print both values? in wei and ethers

Reply:
fixed in 355aa347d608a67055e5f6517e0d1543a579d8ba